dge_files/replogle_preprocessor.py

```diff
@@ -0,0 +1,98 @@
+from typing import List
```

Comment:
Reviewer note -- this is for testing and data generation, will remove before merge.